scripts/photon-docker.sh

```diff
@@ -9,6 +9,7 @@
 echo '> Enabling Docker...'
 
 systemctl enable docker
+reboot
 
 echo '> Done'
```

Comment:
Is this still called even though we add reboot?

Reply:
Yes, this is part of the post-PhotonOS installation and is needed to be able to perform the docker pull

Reply:
Are you also referring to this line?

> echo '> Done'

I.e. after a `reboot`, would that line be executed?

Reply:
That's correct. This particular script will simply enable Docker, reboot and then come back and continue executing the remainder post-deploy scripts